roles/django_app_k8s/templates/rabbitmq.yml.j2

```diff
@@ -0,0 +1,69 @@
+---
+
+kind: Deployment
+apiVersion: v1
+metadata:
+  labels:
+    app.kubernetes.io/name: rabbitmq
+    app.kubernetes.io/instance: "{{ django_app_k8s_instance }}"
+    app.kubernetes.io/version: "3.7"
```

Comment:
See service version remark

Reply:
added and using: `django_app_k8s_rabbitmq_version: "3.10"`